exercises/hello-world/hello_world.go

```diff
@@ -10,6 +10,16 @@ package greeting
 // after you have posted this code to the Exercism site -- reviewers
 // will see that your code can't necessarily be expected to pass the
 // current test suite because it was written to an earlier test version.
+//
+// This is a convention done for Exercism exercises in the Go language track,
```

Comment:
@petertseng take a look at this line. Does it make sense? I wanted to make it clear to people brand new to Go that this entire test versioning setup is a *convention* used by this language track on Exercism and not something used universally in Go. I think that is an easy thing to trip up on since at first glance it really has nothing to do with "Hello World" at all, easy to think it is some language necessity.